products/macos.md

```diff
@@ -25,6 +25,14 @@ auto:
       -   '^Mac\s+OS\s+X\s[\w\s]+\sv?(?P<version>\d{2}(?:\.\d+)+)'
 
 releases:
+-   releaseCycle: "15"
+    codename: "Sequoia"
+    releaseDate: 2024-09-16
+    eol: false
+    latest: '15.0.0'
+    latestReleaseDate: 2024-09-16
+    link: https://support.apple.com/HT213895
```

Comment:
the sequoia support link still does not exist yet.

Reply:
Yeah, it is not available yet.
I’m am waiting it is available to update and merge.